main.go

```diff
@@ -102,6 +104,7 @@ const (
 	defaultLeaseDuration      = 10 // In Seconds
 	defaultRenewDeadline      = 5  // In Seconds
 	defaultRetryPeriod        = 2  // In Seconds
+	componentName             = "submariner-gateway"
```

Comment:
Lets use the names from https://github.com/submariner-io/submariner-operator/blob/ffeb0c063f9aafc0de6fe978d9b5564ed342fe89/pkg/names/names.go

Reply:
Vishal and I were discussing on slack. We can't add a dependency on **submariner-operator** but we can add the component names to **admiral**.